### ISSUE TITLE
Round trip conversion leads to invalid LaTeX

### ISSUE THREAD

**Issue by Smurf-IV** [verybadcat/CSharpMath#57]
**Describe the bug**

**To Reproduce**
Use the following as the source of the LaTeX
`F_n=\frac{5V_m}{\sqrt{g}}`
```
            MathListBuilder mlb = new MathListBuilder(inputLaTeX);
           Trace.Write(mathList.StringValue);
```
Output is this:
`F_{n}=\\atop{0,515V_{m}}{\\Sqrt{g⋅L_{WL}}}`

which is invalid

**Expected behavior**
Source LaTex produces this
![image](https://user-images.githubusercontent.com/2418812/64708274-421a6400-d4ac-11e9-8803-2af69f3c33c9.png)

Round Trip produces this:
![image](https://user-images.githubusercontent.com/2418812/64708678-e3091f00-d4ac-11e9-9fa0-71d33d347421.png)

These images are from http://www.wiris.com/editor/demo/en/developers

**Environment (please complete the following information):**
using CSharpMath.Atoms;
using CSharpMath.Enumerations;
using CSharpMath.Interfaces;
.Net Core 2.2

**Comment by Happypig375** [comment]
`StringValue` is debug-only and is not required to be valid LaTeX. For MathList->LaTeX, use `MathListBuilder.MathListToString`.

**Comment by Happypig375** [comment]
Merged into #58
